src/core_plugins/kibana/public/dashboard/actions/view.js

```diff
@@ -0,0 +1,6 @@
+import { createAction } from 'redux-actions';
+
+export const updateViewMode = createAction('UPDATE_VIEW_MODE', viewMode => viewMode);
```

Comment:
See my comment in the panels actions; all of these action creators don't need to the second argument.